tests/kuttl/supplychain/templates-ytt/01-assert.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 VMware
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#---
+#apiVersion: carto.run/v1alpha1
+#kind: ClusterSupplyChain
+#metadata:
+#  name: responsible-ops---consume-output-of-resources
+#status:
+#  conditions:
+#    - type: TemplatesReady
+#      status: "True"
+#      reason: Ready
+#    - type: Ready
+#      status: "True"
+#      reason: Ready
```

Comment:
Is there a reason this file is commented out?

Reply:
no